client/pages/Dashboard.js

```diff
@@ -39,9 +39,8 @@ class Dashboard extends React.Component {
 
     nprogress.configure({ showSpinner: false });
     nprogress.start();
-    const response = await fetch('/api/users/current/events', { credentials: 'same-origin' });
+    const response = await fetch(' /api/events/getByUser', { credentials: 'same-origin' });
```

Comment:
I just went through the diff of your last commit, and you didn't edit Dashboard.js in that at all.